AstronomyTests/AstronomyTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  AstronomyTests.swift
+//  AstronomyTests
+//
+//  Created by Lydia Zhang on 4/20/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import XCTest
+@testable import Astronomy
+
+struct MockDataLoader: NetworkDataLoader {
+    func loadData(from request: URLRequest, completion: @escaping (Data?, Error?) -> Void) {
+        DispatchQueue.global().asyncAfter(deadline: .now() + 2) {
+            completion(self.data, self.error)
+        }
+    }
+    
+    func loadData(from url: URL, completion: @escaping (Data?, Error?) -> Void) {
+        DispatchQueue.global().asyncAfter(deadline: .now() + 2) {
+            completion(self.data, self.error)
+        }
+    }
+    
+    var data: Data?
+    var error: Error?
+}
```

Comment:
Perhaps this could be in it's own file - would make the test case file a bit cleaner.